sources/tools/Stride.NuGetLoader/Stride.NuGetLoader.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<Project>
-  <Import Project="Sdk.props" Condition="'$(StrideNuGetLoaderWindowsDesktop)' != 'true'" Sdk="Microsoft.NET.Sdk" />
-  <Import Project="Sdk.props" Condition="'$(StrideNuGetLoaderWindowsDesktop)' == 'true'" Sdk="Microsoft.NET.Sdk.WindowsDesktop" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
A warning when building mentioned that `Microsoft.NET.Sdk.WindowsDesktop` is not necessary anymore, this change and the one below deals with that